### PR TITLE
Simplify bridge interface

### PR DESCRIPTION
**Pull request by logan-anderson**
Implements suggestions in https://linear.app/tina/issue/ENG-916/spike-on-simplifying-bridge

cc @jeffsee55

### CODE REVIEW

**Comment by kldavis4** [comment]
@logan-anderson here's my proposal for getting rid of that updateConfig handler from the database: https://github.com/tinacms/tinacms/pull/3842

File: packages/@tinacms/cli/src/next/codegen/index.ts [hunk 1]
```diff
@@ -11,25 +14,61 @@ export class Codegen {
...
     this.queryDoc = queryDoc
     this.fragDoc = fragDoc
+    this.lookup = lookup
+  }
+
+  async writeConfigFile(fileName: string, data: string) {
+    const tinaFolderPath = await this.configManager.getTinaFolderPath(
+      this.configManager.rootPath
+    )
+    const filePath = path.join(tinaFolderPath, GENERATED_FOLDER, fileName)
```
Comment (jeffsee55) [inline packages/@tinacms/cli/src/next/codegen/index.ts:58]:
```suggestion
    const filePath = path.join(this.configManager.generatedFolderPath, fileName)
```

**Review by jeffsee55 — APPROVED** [review]
Small comment but no big deal, looks good

File: packages/@tinacms/cli/src/next/codegen/index.ts (hunk 1 above)
Comment (kldavis4) [reply packages/@tinacms/cli/src/next/codegen/index.ts:58]: @jeffsee55 what about the one on line 68 below it?

Comment (jeffsee55) [reply packages/@tinacms/cli/src/next/codegen/index.ts:58]: Yes sorry same thing in both places.

Comment (logan-anderson) [reply packages/@tinacms/cli/src/next/codegen/index.ts:58]: @jeffsee55 Do we have a generatedFolder path for both the rootPath and the contentRootPath? id like to keep it consistent and leave it as is if we don't

Comment (jeffsee55) [reply packages/@tinacms/cli/src/next/codegen/index.ts:58]: Oh sorry, I misunderstood that. No we don't have that, but would be handy to have as part of the config manager.

**Comment by logan-anderson** [comment]
@kldavis4  and @jeffsee55  I updated the ConfigManager based on @jeffsee55  Feedback.

**Review by kldavis4 — APPROVED** [review]

**Pull request merged**